source/hash_table.c

```diff
@@ -327,7 +327,6 @@ static int s_find_entry1(
     /* If we find a deleted entry, we record that index and return it as our probe index (i.e. we'll keep searching to
```

Comment:
While we're in here, I think the probe index needs to be `size_t` as well.

Reply:
+1 to this.